src/registry/search.rs

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+
+//! Search a semantic convention registry.
+
+use clap::Args;
+
+/// Parameters for the `registry search` sub-command
+#[derive(Debug, Args)]
+pub struct SearchRegistry {
+    /// Git URL of the semantic convention registry
```

Comment:
We should probably uniformly allow "git url" & "local path" or some denoatation of acceptable "location" arguments.  I've opened #31 for that.

Reply:
Thanks